druntime/src/rt/monitor_.d

```diff
@@ -230,7 +230,7 @@ struct Monitor
 
 private:
 
-@property ref shared(Monitor*) monitor(return scope Object h) pure nothrow @nogc
+@property ref shared(Monitor*) monitor(Object h) pure nothrow @nogc
```

Comment:
This signature is correct, the resulting `ref` points into the `Object`, it can be `return scope`

Reply:
you're right